django/db/backends/base/schema.py

```diff
@@ -77,6 +77,7 @@ class BaseDatabaseSchemaEditor:
         "REFERENCES %(to_table)s (%(to_column)s)%(deferrable)s"
     )
     sql_create_inline_fk = None
+    sql_add_field_inline_fk = None
```

Comment:
Good call—I hadn't noticed that. I think I'll go with `sql_create_column_inline_fk` (since `add` is _also_ not consistent with any of the other attribute names).